eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -105,6 +123,27 @@ function List-PullRequests {
   return Invoke-GitHubAPIGet -apiURI $uri
 }
 
+# 
+<#
+.PARAMETER Ref
+Ref to search for
+Pass 'heads/<branchame> ,tags/<tag name>, or nothing
+#>
+function List-References {
```

Comment:
I think we should have a better name for this function as List-References without context is hard to understand as it is very ambiguous with other things. For example reference means something in PS as well. Perhaps `List-GithubSourceReferences` or something like that might be a little clearer.